tchannel-example/src/main/java/com/uber/tchannel/thrift/KeyValueClient.java

```diff
@@ -82,12 +84,14 @@ public static String getValue(
 
         KeyValue.getValue_args getValue = new KeyValue.getValue_args(key);
 
-        ListenableFuture<Response<KeyValue.getValue_result>> future = tchannel.callThrift(
+        ListenableFuture<Response<KeyValue.getValue_result>> future = tchannel
+            .makeSubChannel("keyvalue-service")
+            .callThrift(
                 InetAddress.getLocalHost(),
```

Comment:
Set host port in TChannel is for listen()
Here is for remote host port. This is direct connection. When "peers" are passed to subChannel, peer selection will become effective.